src/client-side-encryption/state_machine.ts

```diff
@@ -352,7 +352,7 @@ export class StateMachine {
     const message = request.message;
     const buffer = new BufferPool();
 
-    const netSocket: net.Socket = new net.Socket();
+    let netSocket: net.Socket;
```

Comment:
Can we move this into the `if` statement and make it const, for clarify?  It isn't used anywhere outside the proxy support if-statement.